ratelimiting/ratelimiting_user.c

```diff
@@ -256,24 +318,29 @@ int main(int argc, char **argv)
     {
         switch (opt) {
             case 'r':
+                printf("rate : %s\n",optarg);
                 rate = strtoi(optarg);
                 break;
             case 'i':
+                printf("iface : %s\n",optarg);
                 ifindex = if_nametoindex(optarg);
                 break;
             case 'v':
+                printf("verbose : %s\n",optarg);
                 if(optarg) {
                     verbosity = strtoi(optarg);
                 }
                 break;
             case 'm':
+                printf("map-name : %s\n",optarg);
                 if(optarg) {
-                    len = get_length(optarg);
-                    strncpy(prev_prog_map, optarg, len);
+                    len = get_length("C:\\leaf\\sys\\fs\\bpf\\xdp_root_pass_array");
```

Comment:
this can be reverted, initially it was all failing when it was optional parameter under long_options[]

Reply:
reverted